extras/backstage/values-kustomization.yaml

```diff
@@ -0,0 +1,27 @@
+---
+apiVersion: kustomize.toolkit.fluxcd.io/v1beta2
+kind: Kustomization
```

Comment:
Why to put values into a separate flux Kust? Seems an overkill to split a single app into 2 Kustomizations